lib/span.js

```diff
@@ -125,7 +125,7 @@ Span.makeEntrySpan = function makeEntrySpan (name, settings, kvpairs) {
   // supply a default in case the user didn't provide a txname string or
   // function that returns a string. if the span is unnamed then let oboe
   // provide "unknown". there is no customTxName function by default.
-  span.defaultTxName = span.name ? 'custom-' + span.name : '';
+  span.defaultTxName = span.name ? 'default-' + span.name : '';
```

Comment:
Minor: we actually sort of agreed in https://swicloud.atlassian.net/browse/AO-8450 that for SDK started traces we'll just default to something like `custom-<top span name>`... does the above line impact that?

Reply:
yes, let me change it back. trying to clean up something that shouldn't have been.